clip.bash

```diff
@@ -85,25 +89,30 @@ cmd_clip() {
     local prompt='Copy password into clipboard for 45 seconds'
     local fzf_cmd="fzf --print-query --prompt=\"$prompt \""
     local rofi_cmd="rofi -dmenu -i -p \"$prompt\""
+    local choose_cmd="choose -m -p \"$prompt\""
 
     if [ -n "$term" ]; then
         fzf_cmd="$fzf_cmd -q\"$term\""
         rofi_cmd="$rofi_cmd -filter \"$term\""
+        choose_cmd="$choose_cmd -o \"$term\""
     fi
     fzf_cmd="$fzf_cmd | tail -n1"
 
-    if [[ $fzf = 1 && $rofi = 1 ]]; then
-        die 'Either --fzf,-f or --rofi,-r must be given, not both'
+    if [[ ($fzf = 1 && $rofi = 1) || ($fzf = 1 && $choose = 1) || ($rofi = 1 && $choose = 1) ]]; then
```

Comment:
Not your fault but this will become ridiculous if we add more to it 😅

Reply:
**Suggestion:** This could work to simplify it:
```bash
if (( fzf + rofi + choose > 1 )); then
```

Reply:
Agreed, I was thinking of something like that too. No need to do that in this PR though.

Reply:
Agreed too, if the merge happens, this could be the time to introduce a new `--provider=[fzf|rofi|choose]` argument that will simplify the overall logic for these providers.